### DISCUSSION TITLE
Welcome to misskey Discussions!

### DISCUSSION THREAD

**Discussion by sousuke0422** [TeamOrangeServer/misskey#56] [General]
日本語で(ry
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yupix** [comment]
yupixと申します。

基本的にデザイン部分を担当しているのは私です。
たまにデザイン以外の部分もやらないこともないです。

**Comment by yupix** [comment]
ブラー系とかと一緒に作ってみたから見た目のレビューよろしく、あとデザイン系はもう基本akiは弄らないってことでいい?
